<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result</class>
 <widget class="QWidget" name="Result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <widget class="QLabel" name="Result_DB1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Result_DB2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>241</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Result_DB3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>381</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Result_DB3_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>381</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
